run-test.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/ruby
```

Comment:
If this file provides anything in terms of testing, it should be done with rspec tests in the `spec` directory. Otherwise, just delete it.

I guess you could call it documentation? rename it to `example.rb` or something then.

Reply:
Yes. Renamed to `example.rb`